packages/x-components/src/__tests__/utils.ts

```diff
@@ -174,7 +176,9 @@ export function getMockedPlatformAdapter(
     /* eslint-disable @typescript-eslint/no-non-null-asserted-optional-chain */
     identifierResults: getMockedAdapterFunction(responseFeatures?.identifierResults!),
     nextQueries: getMockedAdapterFunction(responseFeatures?.nextQueries!),
+    popularSearches: getMockedAdapterFunction(responseFeatures?.popularSearches!),
     querySuggestions: getMockedAdapterFunction(responseFeatures?.querySuggestions!)
+    /* eslint-enable @typescript-eslint/no-non-null-asserted-optional-chain */
```

Comment:
remove this line

Reply:
By removing it, the following lines will have the `@typescript-eslint/no-non-null-asserted-optional-chain` disabled because this rule has been disabled using `eslint-disable` instead `eslint-disable-next-line`.